packages/@okta/courage-for-signin-widget/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "courage-for-signin-widget",
+  "version": "1.0.0",
+  "description": "generates a Courage version for signin-widget",
+  "main": "src/CourageForSigninWidget.js",
+  "scripts": {
+    "build": "webpack"
+  },
+  "author": "",
+  "license": "MIT",
+  "dependencies": {
+    "@okta/courage": "4.4.0-beta.2818bde",
```

Comment:
following my comment above, we should try use release versions here instead of beta (beta can be use internally). Or we can talk about it and change it as well. Technically it doesn't matter much, but we decided that long time ago just to keep our open source repo using release versions

Reply:
good point and I'm not aware that we like our open source repo to use release version.
especially for courage, turns out we didn't make very clear when / what to be release.
will make change to use release version and like to chat more about it.

Reply:
@mauriciocastillosilva-okta yeah, added to uicore office hour topic.